src/enums.h

```diff
@@ -314,9 +314,37 @@ inline bool operator<( const tripoint &a, const tripoint &b )
 
 static constexpr tripoint tripoint_min { INT_MIN, INT_MIN, INT_MIN };
 static constexpr tripoint tripoint_zero { 0, 0, 0 };
+static constexpr tripoint tripoint_max{ INT_MAX, INT_MAX, INT_MAX };
 
 static constexpr point point_min{ tripoint_min.x, tripoint_min.y };
 static constexpr point point_zero{ tripoint_zero.x, tripoint_zero.y };
+static constexpr point point_max{ tripoint_max.x, tripoint_max.y };
+
+/** Checks if given tripoint is inbounds of given min and max tripoints using given clearence **/
+inline bool generic_inbounds( const tripoint &p,
+                              const tripoint &p_min = tripoint_min,
```

Comment:
A default for min and max seems pointless. What would be the meaning of `generic_inbounds( p )`? What range does that check?

Reply:
You are correct. It is still WIP, but I forgot to add tag (fixed it now).